dev-utils/build.js

```diff
@@ -139,67 +141,116 @@ function getWebpackEnv(env = 'development') {
   }
 }
 
-function getWebpackConfig(bundleType, packageType) {
+function isProduction(bundleType) {
   const isEnvProduction = [
     NODE_PROD,
     NODE_ESM_PROD,
     BROWSER_PROD,
     BROWSER_ESM_PROD
   ].includes(bundleType)
 
-  const config = {
+  return isEnvProduction
+}
+
+function getCommonWebpackConfig(bundleType, packageType) {
+  const isEnvProduction = isProduction(bundleType)
+  const mode = isEnvProduction ? 'production' : 'development'
+
+  return {
+    devtool: isEnvProduction ? 'source-map' : false,
+    mode,
     stats: {
       colors: true,
-      warnings: false
+      assets: true,
+      modules: false
```

Comment:
Wasn't `modules:false` the reason we didn't get the right bundle analyzer report?

Reply:
Nope, it was just hidden inside the flag on UI. modules: false will disable the file level comment that we get on stdout.